_examples/aks/main.tf

```diff
@@ -0,0 +1,49 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+    }
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "2.42"
+    }
+    helm = {
+      source  = "hashicorp/helm"
+      version = "2.0.1"
```

Comment:
Should have a `=>` constraint here and in the one above since this is an example.

Reply:
Do you mean `version = ">= 2.0.1"`? I haven't used version constraints much. You can also click "insert a suggestion" to help me figure out exactly where to put this. Thanks!

Reply:
Yes sorry that's what I mean. Just so it will pull down the latest version of the provider without us having to go back and update the docs all the time. This way we would only have to go back and change it when we have another major version bump.

The syntax is [here](https://www.terraform.io/docs/configuration/version-constraints.html.

Reply:
For the cloud providers (like azurerm), I was going to leave the versions static so we don't have to update them until we want to. Potentially it would keep working for years. For Kubernetes and Helm though, I'm ok with these auto-updating, since we'll know about any breaking changes.

Reply:
Yep that makes sense!